udata_front/theme/gouvfr/assets/less/content/not-enlarged-links.less

```diff
@@ -0,0 +1,19 @@
+.not-enlarged {
+    position: relative;
+    z-index: 2;
+}
+
+.fr-enlarge-link .not-enlarged a {
+    background-image: var(--underline-img),var(--underline-img);
+    &:hover {
+        // from DSFR [href] style, we cant :extend it anymore
```

Comment:
Can you move the DSFR related part to a [detached ruleset](https://lesscss.org/features/#detached-rulesets-feature) ? This would allow us to put it in [variables](https://github.com/etalab/udata-front/blob/1b4a7d9019986389fb80fc9c87a24737c09a9609/udata_front/theme/gouvfr/assets/less/variables.less#L120) and improve discoverability for future uses.

Reply:
I pushed a fix, let me know if it's what you expected